### PR TITLE
Set up FDC page

### PR DESCRIPTION
**Pull request by cehsu**
[BLOCKED] : We may not need this page per @goldenmeanie's insights.

These changes set up the FDC page. The updated schema is intended to be merged into this, as there were several errors unrelated to this chapter that looked like they would be resolved by that update. I'll mention that on the description.

TODO:
- [x] merge updated config into this
- [x] move helpers and remove React import from form.js
- [ ] write tests

([Additional ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9109))

screen shot 2018-04-13 at 3 32 45 pm
screen shot 2018-04-13 at 3 32 37 pm

### CODE REVIEW

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cehsu Hi, Claire, this is not the content I provide for the claims increase FDC page. Please see this ticket for the correct copy: 
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9950

**Comment by cehsu** [comment]
Thanks @peggygannon, updated.
screen shot 2018-04-13 at 4 33 38 pm
screen shot 2018-04-13 at 4 34 55 pm
screen shot 2018-04-13 at 4 35 13 pm

**Review by peggygannon — APPROVED** [review]

File: src/js/disability-benefits/526EZ/config/form.js [hunk 1]
```diff
@@ -78,593 +84,109 @@ const formConfig = {
   migrations: [],
   savedFormMessages: {
     notFound: 'Please start over to apply for disability claims increase.',
-    noAuth: 'Please sign in again to resume your application for disability claims increase.'
+    noAuth:
+      'Please sign in again to resume your application for disability claims increase.'
```
Comment (cvalarida) [inline src/js/disability-benefits/526EZ/config/form.js]: Can we move this back up to the line on which the property is defined?

File: src/js/disability-benefits/526EZ/config/form.js [hunk 2]
```diff
@@ -78,593 +84,109 @@ const formConfig = {
...
-          path: 'chapter-one/page-one',
-          uiSchema: {},
-          schema: {
-            type: 'object',
-            properties: {}
-          },
-        }
-      }
-    },
-    chapterTwo: {
```
Comment (cvalarida) [inline src/js/disability-benefits/526EZ/config/form.js]: Um...I'm guessing you didn't intend to nuke all the other chapters?

File: src/js/disability-benefits/526EZ/config/form.js [hunk 3]
```diff
@@ -78,593 +84,109 @@ const formConfig = {
...
+            noRapidProcessing: {
+              'ui:title':
+                'Do you want to apply using the Fully Developed Claim program?',
+              'ui:widget': 'yesNo',
+              'ui:options': {
+                yesNoReverse: true,
+                labels: {
+                  Y: 'Yes, I have uploaded all my supporting documents.',
+                  N:
+                    'No, I have some extra information that I will submit to VA later.'
```
Comment (cvalarida) [inline src/js/disability-benefits/526EZ/config/form.js]: Is it a VS Code setting to move long strings down to their own line? It only moves the whole string one character to the left, so it's not actually saving much horizontal space.

File: src/js/disability-benefits/526EZ/config/form.js [hunk 4]
```diff
@@ -78,593 +84,109 @@ const formConfig = {
...
+                  Learn more about the FDC program
+                </a>.
+              </div>
+            ),
+            noRapidProcessing: {
+              'ui:title':
+                'Do you want to apply using the Fully Developed Claim program?',
+              'ui:widget': 'yesNo',
+              'ui:options': {
+                yesNoReverse: true,
```
Comment (cvalarida) [inline src/js/disability-benefits/526EZ/config/form.js]: Oh interesting. What does this do?

**Review by cvalarida — COMMENTED** [review]
Looks like this is still a WIP, but once you bring back the other chapters, it looks good--just a couple of nits.

**Comment by cehsu** [comment]
UPDATE: These changes are blocked as we may not need this page per @goldenmeanie's insights.

**Review by cvalarida — APPROVED** [review]
This may not need to be merging as you said, but in case it is still necessary, I'll go ahead and approve it. I anticipate it'll be a fun merge conflict to resolve, though. :grimacing:

**Pull request merged**